lldb/include/lldb/Target/StackID.h

```diff
@@ -57,15 +57,17 @@ class StackID {
     return *this;
   }
 
-  bool IsCFAOnStack() const { return m_cfa_on_stack; }
+  /// Check if the CFA is on the stack, or elsewhere in the process, such as on
+  /// the heap.
+  bool IsCFAOnStack(Process &process) const;
+
+  /// Determine if the first StackID is "younger" than the second.
+  static bool IsYounger(const StackID &lhs, const StackID &rhs);
```

Comment:
Prob just me, but I'm not completely clear on what "younger" means here.  If `foo()` calls `bar()` and we do a backtrace, is the `bar()` stack frame younger than the `foo()` stack frame.  Younger meaning most recently created stack frame.

Reply:
Yes, that's correct.

I don't find the term "younger" entirely intuitive either, but I followed the existing use of the term. Do you have an alternative suggestion?

Reply:
Not offhand.  Maybe we can stick a definition like above in the comment.